### PR TITLE
[Windows] add check driver version in test case check_inbox_driver

### LINKED ISSUE #345: [New Feature][Windows] improve test case check_inbox_driver
##### SUMMARY

Improve test case check_inbox_driver to check if there is inbox driver firstly, no using parameters to determine.
Then get inbox driver info if there are inbox drivers.

##### ISSUE TYPE
- Feature Idea

##### ENVIRONMENT / OS

Windows

### PR DESCRIPTION
**Pull request by Tomorrow9**
Signed-off-by: Diane Wang <dianew@vmware.com>
Fixes #345

### CODE REVIEW

**Comment by Tomorrow9** [comment]
```
VM information:
+-----------------------------------------------------------------------------+
| Name                      | test_vm                                         |
+-----------------------------------------------------------------------------+
| IP                        |                                   |
+-----------------------------------------------------------------------------+
| Guest OS Distribution     | Microsoft Windows 11 Enterprise Insider Preview |
|                           | 10.0.25276.0 64-bit                             |
+-----------------------------------------------------------------------------+
| Hardware Version          | vmx-20                                          |
+-----------------------------------------------------------------------------+
| VMTools Version           |                                                 |
+-----------------------------------------------------------------------------+
| Config Guest Id           | windows11_64Guest                               |
+-----------------------------------------------------------------------------+
| GuestInfo Guest Id        |                                                 |
+-----------------------------------------------------------------------------+
| GuestInfo Guest Full Name |                                                 |
+-----------------------------------------------------------------------------+
| GuestInfo Guest Family    |                                                 |
+-----------------------------------------------------------------------------+
| GuestInfo Detailed Data   |                                                 |
+-----------------------------------------------------------------------------+

Test Results (Total: 2, Passed: 2, Elapsed Time: 00:28:38)
+--------------------------------------------------------+
| Name                              | Status | Exec Time |
+--------------------------------------------------------+
| deploy_vm_efi_paravirtual_vmxnet3 | Passed | 00:19:09  |
| check_inbox_driver                | Passed | 00:09:06  |
+--------------------------------------------------------+
```

**Review by 123lzxm — DISMISSED** [review]
LGTM, thanks.

File: docs/TESTCASE_DEV_GUIDE.md [hunk 1]
```diff
@@ -21,64 +22,78 @@
...
 
 2. In the "block" part, write the tasks of test case execution and verification.
 
-3. In the "block" part, use this task "common/print_test_result.yml" to set test case result as below:
-* "No Run": test case requirements or dependency not meet, 
-* "Failed": test case verification failed,
-* "Passed": test case passed.
+3. In the "block" part, use this task "common/skip_test_case.yml" to set test case result as below:
+* Blocked: Test case dependency is not meet, e.g. no VMtools or no VC for GOSC testing
+* Not Supported: Tested function is not supported by the OS.
```
Comment (keirazhang) [inline docs/TESTCASE_DEV_GUIDE.md:90]: Test case could be not supported by either the guest OS or the VM's config (e.g. guest id, hardware version).

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**